pkg/runner/local_runner.go

```diff
@@ -70,10 +72,18 @@ func (r *localRunner) Run(ctx context.Context, request *runner.RunRequest) (*run
 	if len(request.Arguments) < 1 {
 		return nil, status.Error(codes.InvalidArgument, "Insufficient number of command arguments")
 	}
-	cmd := exec.CommandContext(ctx, request.Arguments[0], request.Arguments[1:]...)
-	// TODO: Convert WorkingDirectory and TemporaryDirectory to use
-	// platform specific path delimiters.
-	cmd.Dir = filepath.Join(r.buildDirectoryPath, request.InputRootDirectory, request.WorkingDirectory)
+	var cmd *exec.Cmd
+	if r.chrootIntoInputRoot {
```

Comment:
Could you add a comment here to explain why the `/usr/bin/env` is added? If you feel like filing a bug against upstream Go, please do! Feel free to include a link to that bug report in the comment.